pkg/codegen/internal/test/sdk_driver.go

```diff
@@ -196,7 +220,9 @@ func TestSDKCodegen(t *testing.T, opts *SDKCodegenOptions) { // revive:disable-l
 	testDir := filepath.Join("..", "internal", "test", "testdata")
 
 	for _, tt := range sdkTests {
-		t.Run(tt.Description, func(t *testing.T) {
+		t.Run(tt.Directory, func(t *testing.T) {
```

Comment:
Note: I'm finding it easier to type `go test -run TestGeneratePackage/output-funcs` by folder name, not description.